### ISSUE TITLE
Security workshop provisioner fails due to timeout on "wait for pre-reqs install async"

### ISSUE THREAD

**Issue by ptoal** [ansible/workshops#899]
##### SUMMARY
Security Provisioner can fail due to async yum install with too short a timeout.

##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION
```paste below
ansible 2.9.9
  config file = /Users/ptoal/.ansible.cfg
  configured module search path = ['/Users/ptoal/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/ptoal/.virtualenvs/ansible/lib/python3.8/site-packages/ansible
  executable location = /Users/ptoal/.virtualenvs/ansible/bin/ansible
  python version = 3.8.2 (default, Feb 27 2020, 19:58:50) [Clang 11.0.0 (clang-1100.0.33.17)]
```

##### OS / ENVIRONMENT
MacOS 10.15.4

##### PLAYBOOK SHORT OUTPUT

```paste below
TASK [package pre-reqs are installed] ***************************************************************************************************************
Wednesday 03 June 2020  09:17:43 -0400 (0:00:00.020)       0:07:17.758 ********
changed: [ptoal-sec-student1-snort]

TASK [wait for pre-reqs install async] **************************************************************************************************************
Wednesday 03 June 2020  09:17:44 -0400 (0:00:01.234)       0:07:18.993 ********
FAILED - RETRYING: wait for pre-reqs install async (10 retries left).
FAILED - RETRYING: wait for pre-reqs install async (9 retries left).
FAILED - RETRYING: wait for pre-reqs install async (8 retries left).
FAILED - RETRYING: wait for pre-reqs install async (7 retries left).
FAILED - RETRYING: wait for pre-reqs install async (6 retries left).
FAILED - RETRYING: wait for pre-reqs install async (5 retries left).
FAILED - RETRYING: wait for pre-reqs install async (4 retries left).
FAILED - RETRYING: wait for pre-reqs install async (3 retries left).
FAILED - RETRYING: wait for pre-reqs install async (2 retries left).
FAILED - RETRYING: wait for pre-reqs install async (1 retries left).
fatal: [ptoal-sec-student1-snort]: FAILED! => changed=false
  ansible_job_id: '515224648805.12578'
  attempts: 10
  finished: 0
  started: 1
```

**Comment by cigamit** [comment]
Why are we even doing an async on that?  Its not saving any time, as the very next task is the async_status task to wait for it to complete.

**Comment by ptoal** [comment]
I wondered that too, but that's a question for @IPvSean .  I just wanted to make the playbook work again with the minimum disruption.

**Comment by IPvSean** [comment]
I would prefer not using async it seems not Ansible to me... and it causes
issues with aws api requests , this is all Adams work 😛

**Comment by ptoal** [comment]
Updated the PR to pull out the async bit.  I presumed it had some future purpose, but I now suspect it was just copy/paste from another playbook.

Tests don't pass, but I used these changes successfully today.

**Comment by ptoal** [comment]
Fixed by PR #904
